main_test.go

```diff
@@ -24,3 +19,15 @@ func Test_Main_Version(t *testing.T) {
 
 	assert.Contains(t, out, "dev")
 }
+
+func setArgs(t *testing.T, args ...string) {
+	t.Helper()
+
+	old := os.Args
```

Comment:
the var name `old` seems to be a bit confusing here, can we rename it to something better?